bundled/tool/find_ruff_binary_path.py

```diff
@@ -32,4 +39,4 @@ def find_ruff_binary_path() -> Optional[Path]:
 if __name__ == "__main__":
     ruff_binary_path = find_ruff_binary_path()
     if ruff_binary_path:
-        print(os.fsdecode(str(ruff_binary_path)), flush=True)
+        print(os.fsdecode(ruff_binary_path), flush=True)
```

Comment:
I think the call to `str` before calling `os.fsdecode` is just wrong? But how would I know, I don't know nothing of Python

Reply:
Yeah, I think that's correct. `print(<pathlib.Path>)` would use the `pathlib.Path.__str__` method, so the `str` call here is redundant.